server/scsynth/CMakeLists.txt

```diff
@@ -99,6 +99,7 @@ if (AUDIOAPI STREQUAL jack)
 	include_directories(${JACK_INCLUDE_DIRS})
 elseif (AUDIOAPI STREQUAL portaudio)
 	list(APPEND scsynth_sources SC_PortAudio.cpp)
+	list(APPEND scsynth_sources ${CMAKE_SOURCE_DIR}/common/SC_PaUtils.cpp)
```

Comment:
this can be combined with the previous line, check the documentation for `list(APPEND`